### ISSUE TITLE
My work for PR #48 has been removed ?

### ISSUE THREAD

**Issue by Anirban-1490** [prathimacode-hub/Treksy#52]
My work got merged at PR #48 and now the work got removed from contact.css and contact.html . (contact form backend)

@prathimacode-hub is this supposed to happen ??

**Comment by prathimacode-hub** [comment]
I don't think so it is removed. As I could see both the files. I only merge PR's. @Anirban-1490
